data/spells/scripts/support/levitate.lua

```diff
@@ -0,0 +1,36 @@
+function onCastSpell(creature, variant)
+	local currentPosition = creature:getPosition()
+	local destinationPosition = creature:getPosition()
+	destinationPosition:getNextPosition(creature:getDirection())
```

Comment:
I totally agree with that @raymondtfr, this function has made me scratch my head alot of times...
But it does not matter in this PR, if you wanna get that change done before this PR is done then do that, otherwise do it "the right way" as I told you above, you change the variable by setting nextPosition?.z to another value.

But IMO we should update that function aswell as all the scirpts that uses that function & inform @ninjalulz so he can update orts.